src/components/ImageCarousel.tsx

```diff
@@ -1,37 +1,51 @@
 import React, { ReactElement } from 'react'
+import { Keyboard, useWindowDimensions } from 'react-native'
 import ImageViewer from 'react-native-image-zoom-viewer'
 import { Pagination } from 'react-native-snap-carousel'
 import styled from 'styled-components/native'
 
 import { ImagesType } from '../constants/endpoints'
 import { COLORS } from '../constants/theme/colors'
 
-const ImageView = styled.View`
-  height: 35%;
+const ImageView = styled.View<{ height: number }>`
+  height: ${({ height }) => height}px;
 `
-const StyledImage = styled.Image`
-  height: 100%;
+const StyledImage = styled.Image<{ height: number; minimized: boolean }>`
+  height: ${({ height }) => height}px;
+  ${({ minimized }) =>
+    minimized &&
+    `
+    aspect-ratio: 1.5;
```

Comment:
`resizeMode='center'` did only work on android properly, so i used this way to center and set a nice aspectRadio

Reply:
Perhaps you could add a comment `Center image` if that is the purpose